docs/UserGuide.md

```diff
@@ -42,6 +42,12 @@ HealthSync is a **powerful desktop application designed specifically for clinic
 
    ![Ui](images/Ui_v1.3.1.jpg)
 
+
+Here are the icons you will see throughout this User Guide and what they mean:
+- :bulb: Tips and tricks that will help you get the most out of the app
```

Comment:
Should we rephrase `app` as `HealthSync` instead.

Reply:
Good call, have modified it.